### ISSUE TITLE
Response to upcoming redefinition of SI base units

### ISSUE THREAD

**Issue by dmcclean** [bjornbm/dimensional#206]
What is our planned response to the coming redefinition of the SI base units?

Under the new system, some derived units which were formerly experimentally determined will now have exact values, and vice versa.

See https://en.wikipedia.org/wiki/2019_redefinition_of_SI_base_units#Impact_on_reproducibility for a summary of those changes.

**Comment by bjornbm** [comment]
I'll look into it! What are your thoughts?

**Comment by AtelierSnek** [comment]
I would suggest simply using the stated values as per the propositions (which are in effect as of May 20th by the way). The main issues with the re-definitions as I see it as that we're still having trouble getting accurate enough measurements to use them, however they have been ratified, so the library should assume that they're correct and usable.

**Comment by dmcclean** [comment]
I agree. I have some changes that I plan to make next week, I will do these changes as well if no one beats me to it.
